fantasycourt.nl/content/praktische_info/aanmelden/_index.nl.md

```diff
@@ -0,0 +1,61 @@
+
+---
+title: Standhouders
+header-image:
+  url: images/FC_crew.jpg
+  position-x: 50%
+  position-y: 46%
+og:
+  description: "Om Fantasy Court de beste versie ooit te kunnen geven zijn we op zoek naar mensen die de ervaring op hun eigen manier kunnen verrijken. Standhouders, acts, workshops, demo's, LARP-verenigingen, of iets anders dat past!"
+---
+
+{{< message >}}
+  _Handelaren, edellieden en heren van de gunsten,_ \
+  _Uw aanwezigheid is gewenst, met uw waren en uw kunsten_
+{{< /message >}}
+
+# Standhouders Editie 2025
+Om Fantasy Court nog beter te maken zijn wij altijd op zoek naar meer mensen die ons evenement kunnen verrijken. Wij bieden jullie een geweldig, enthousiast publiek dat komt voor de sfeer en de gespecialiseerde zaken. Want waar in Nederland vind je nou een festival dat zich vooral met LARP en gerelateerde zaken bezighoudt? Als je onze visie deelt kan je je inschrijven op [het inschrijfformulier](https://docs.google.com/forms/d/e/1FAIpQLSdgNXPGttfaqPzkXfZpr1McJyhMBjCHTCpDKVrfEfgqgnLUFQ/viewform?usp=sf_link). Heb je vragen? Neem dan [contact](mailto:standhouders@fantasycourt.nl) met ons op.
+ 
+Om vast te stellen wat we bieden en verwachten hebben we een inschrijfreglement opgesteld. Dit kan je [hier vinden.](https://docs.google.com/document/d/1lWsQuVgl0SjMx5kb9iwUHwva5-iMHPq9/edit?usp=sharing&ouid=118033485815882455862&rtpof=true&sd=true)
```

Comment:
Waarom is dit een extern document? Dit is info die je prima op een subpagina op de website kan neerzetten.